proto/vtctldata.proto

```diff
@@ -1022,6 +1022,84 @@ message LookupVindexExternalizeResponse {
   bool workflow_deleted = 1;
 }
 
+message MigrateCreateRequest {
+  // The necessary info gets passed on to each primary tablet involved
+  // in the workflow via the CreateVReplicationWorkflow tabletmanager RPC.
+  string workflow = 1;
+  string source_keyspace = 2;
+  string target_keyspace = 3;
+  string mount_name = 4;
+  repeated string cells = 5;
+  repeated topodata.TabletType tablet_types = 6;
+  tabletmanagerdata.TabletSelectionPreference tablet_selection_preference = 7;
+  bool all_tables = 8;
+  repeated string include_tables = 9;
+  repeated string exclude_tables = 10;
+  // SourceTimeZone is the time zone in which datetimes on the source were stored, provided as an option in MoveTables
+  string source_time_zone = 11;
+  // OnDdl specifies the action to be taken when a DDL is encountered.
+  string on_ddl = 12;
+  // StopAfterCopy specifies if vreplication should be stopped after copying.
+  bool stop_after_copy = 13;
+  // DropForeignKeys specifies if foreign key constraints should be elided on the target.
+  bool drop_foreign_keys = 14;
+  // DeferSecondaryKeys specifies if secondary keys should be created in one shot after table copy finishes.
+  bool defer_secondary_keys = 15;
+  // Start the workflow after creating it.
+  bool auto_start = 16;
+  // NoRoutingRules is set to true if routing rules should not be created on the target when the workflow is created.
+  bool no_routing_rules = 17;
```

Comment:
Any reason not to embed a MoveTablesCreateRequest and then add the mount_name? Otherwise we'll have to keep these things in sync.